gwtmockito/src/main/java/com/google/gwtmockito/impl/StubGenerator.java

```diff
@@ -61,6 +62,14 @@
     STUB_METHODS.put(
         new ClassAndMethod(InputElement.class, "as"),
         new ReturnMockStubMethod(InputElement.class));
+	// URL.encodeQueryStringImpl
+	STUB_METHODS.put(
+		new ClassAndMethod(URL.class, "encodeQueryStringImpl"),
+		new ReturnStringStubMethod("encodeQueryStringImpl"));
+	// URL.encodePathSegmentImpl
+	STUB_METHODS.put(
+		new ClassAndMethod(URL.class, "encodePathSegmentImpl"),
```

Comment:
Add tests for these too. Is there another method you call that causes these impl methods to break under the hood?